seed/views/v3/organizations.py

```diff
@@ -516,6 +518,16 @@ def warn_bad_units(kind, unit_string):
         if geocoding_enabled != org.geocoding_enabled:
             org.geocoding_enabled = geocoding_enabled
 
+        # Update property_display_field option
+        property_display_field = posted_org.get('property_display_field', True)
```

Comment:
```suggestion
        property_display_field = posted_org.get('property_display_field', '')
```
or maybe None, then don't update if None?

Reply:
Updated to have the correct default "address_line_1", thanks for the catch.